dagmc_stats.py

```diff
@@ -201,3 +201,90 @@ def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
         t_a_r.append(top/bottom)
 
     return t_a_r
+
+def get_area_triangle(my_core, meshset):	
+    """
+    Gets the triangle area (according to the equation: sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2)
```

Comment:
Maybe add a short note that this is Heron's formula to give the user more detail if they are unfamiliar with it and where it comes from.